src/components/Spinner/Spinner.tsx

```diff
@@ -3,7 +3,7 @@ import classNames from 'classnames';
 import { PALMETTO_BRAND_COLOR_VALUES, PALMETTO_COLORS } from '../../lib/tokens';
 import styles from './Spinner.module.scss';
 
-interface Props {
+interface SpinnerProps {
```

Comment:
Hey @JCarmine I am trying to boyscout these as I go. I know it is not ideal, but yes we want to move away from the non-descriptive props since this actually gets exported to a TS type and it is better to have the be specific.